source/math/FloatMatrix.ooc

```diff
@@ -113,9 +116,9 @@ FloatMatrix : cover {
 	// Creates a copy of the current matrix.
 	// </summary>
 	// <returns>Return a copy of the current matrix.</returns>
-	copy: func@ -> This {
-		result := This new(this dimensions)
-		memcpy(result elements data, this elements data, this dimensions area * Float size)
+	copy: func -> This {
+		result := This new(this _elements copy(), this dimensions)
+		this free(Owner Callee)
```

Comment:
It probably is safe to change it to an either or in a later PR that focuses on in-place updates when owner is set to callee.